manager/src/test/java/com/redhat/developer/manager/ConnectorsServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package com.redhat.developer.manager;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.developer.infra.dto.ConnectorStatus;
+import com.redhat.developer.manager.models.Connector;
+import com.redhat.developer.manager.requests.ConnectorRequest;
+import com.redhat.developer.manager.utils.DatabaseManagerUtils;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class ConnectorsServiceTest {
+
+    @Inject
+    ConnectorsService connectorsService;
+
+    @Inject
+    DatabaseManagerUtils databaseManagerUtils;
+
+    @BeforeEach
+    public void cleanUp() {
+        databaseManagerUtils.cleanDatabase();
```

Comment:
Any particular reason you needed to add this over using `@TestTransaction`?

Reply:
@robpblake yes, `@TestTransaction` is pretty dangerous. See here https://github.com/quarkusio/quarkus/issues/15436 . In particular we have tests with restassured -> the transaction is gone and it is not rolled back when the test is over. This is why we have to clean the database before each test. 
If we had only this `ConnectorsServiceTest` we could remove this `@BeforeEach` and add `@TestTransaction` to all the tests, but since we have other tests that leave data in the database the `@BeforeEach` approach is more safe imo. 

Unless we add also a 
```
    @AfterEach
    public void cleanUp() {
        databaseManagerUtils.cleanDatabase();
```
in the restassured test classes or a 
```
    @BeforeAll
    public void cleanUp() {
        databaseManagerUtils.cleanDatabase();
```
in this class we can not use `@TestTransaction`. We might give a try to this https://github.com/radcortez/flyway-junit5-extensions plugin that should clean up the db after each test. 
My main concern is about how do we pass this information to those who are not 100% familiar with @TestTransaction and the testing framework that does not start a new fresh database after each test. wdyt?

Reply:
Your solution works for me 👍  

I'd say let's just add a bit of info into the `DatabaseManagerUtils` class as Javadoc or comments as to why it exists, capturing your points ^. That way when new team members come along, they don't have to ask the same question as me.

Reply:
Makes sense, done